docs/core/tutorials/with-visual-studio.md

```diff
@@ -103,6 +103,7 @@ Enhance the application to prompt the user for their name and display it along w
    This code displays a prompt in the console window and waits until the user enters a string followed by the <kbd>Enter</kbd> key. It stores this string in a variable named `name`. It also retrieves the value of the <xref:System.DateTime.Now?displayProperty=nameWithType> property, which contains the current local time, and assigns it to a variable named `date` (`currentDate` in Visual Basic). And it displays these values in the console window. Finally, it displays a prompt in the console window and calls the <xref:System.Console.ReadKey(System.Boolean)?displayProperty=nameWithType> method to wait for user input.
 
    The `\n` (or `vbCrLf` in the Visual Basic code) represents a newline character.
```

Comment:
This line is no longer needed if we change all 3 instances of \n or vbCrLf to Environment.NewLine.